src/js/environment/environment.ts

```diff
@@ -23,6 +23,10 @@ import { osVersion } from './os-version';
 import { memoize } from 'utils/underscore';
 import type { GenericObject } from 'types/generic.type';
 
+declare global {
```

Comment:
can we move this here https://github.com/jwplayer/jwplayer/blob/master/src/js/types/globals.d.ts